### PR TITLE
build: remove unused deps and update node version refs: SDKUI-6

### PR DESCRIPTION
**Pull request by CerDennis**

### CODE REVIEW

**Review by beawar — COMMENTED** [review]
The file .nvmrc needs to be updated as well

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-v16.13
\ No newline at end of file
+v18.12
```
Comment (beawar) [inline .nvmrc]: We can use only v18 so that we are not strictly linked to a minor I think

File: package.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
     "url": "git+ssh://git@github.com:Zextras/carbonio-ui-sdk.git"
   },
   "engines": {
-    "node": "v16",
+    "node": "v18",
     "npm": "v8"
```
Comment (beawar) [inline package.json]: Why the update of npm was reverted? Shouldn't we update also npm version with node?

File: package.json
Comment (beawar) [inline package.json:1]: Have a check also on these dependencies. I'm not sure whether they are needed:
- @babel/runtime
- child-process (should be a node build in)
- fs (should be a node build in)

File: package.json (hunk 2 above)
Comment (CerDennis) [reply package.json]: Yes with the version now using 18.19.0 we need to update to v10, but with previous version 18.12.0 it use v8 still.

File: package.json
Comment (CerDennis) [reply package.json:1]: Checked and removed

**Review by nubsthead — APPROVED** [review]

**Review by beawar — APPROVED** [review]

**Review by lisaparma — APPROVED** [review]

**Pull request merged**